flutter/lib/resources/utils.dart

```diff
@@ -1,11 +1,13 @@
+import 'package:mlperfbench/resources/resource_manager.dart';
```

Comment:
I think it would be better to move `Resource` definition into a separate file.
It feels wrong to me to import such unrelated file just for one small data class.

Reply:
ok